docs/data/xrootd-authorization.md

```diff
@@ -0,0 +1,141 @@
+Configuring XRootD Authorization
+================================
+
+There are several authorization options in XRootD available through the security plugins. 
+In this document, we will cover the security option supported in the OSG:
+
+-   [`xrootd-lcmaps`](#recommended-security-option-xrootd-lcmaps-authorization): Using the LCMAPS callout to use
+    X509-based authentication
+
+Note: On the data nodes, the files will actually be owned by unix user `xrootd` (or other daemon user), not as the user
+authenticated to, under most circumstances. 
+XRootD will verify the permissions and authorization based on the user that the security plugin authenticates you to,
+but, internally, the data node files will be owned by the `xrootd` user.
+
+#### Authorization file
+
+In order to add security to your cluster you will need to add "auth\_file" on the your data server node. 
+Create `/etc/xrootd/auth_file` :
+
+```file
+# This means that all the users have read access to the datasets 
+u * %RED%/data/xrootdfs%ENDCOLOR% lr
+
+# This means that all the users have full access to their private dirs 
+u = %RED%/data/xrootdfs/%ENDCOLOR%@=/ a
+
+# This means that this privileged user can do everything 
+# You need at least one user like that, in order to create the 
+# private dir for each user willing to store his data in the facility 
+u xrootd %RED%/data/xrootdfs%ENDCOLOR% a 
+```
+
+Here we assume that your storage path is `/data/xrootdfs` (same as in the
+previous example).
```

Comment:
Everything from this line to the top of the `Authorization file` section seems like a premature example. We should describe the following beforehand:

* An introduction (`The XRootD authorization file is where you configure access permissions for your XRootD installation....`)
* File location (let's just use `/etc/xrootd/auth_file`) and permissions
* File format (the table)

Then it makes sense to have each example as their own sub-section. Or if @matyasselmeci thinks it's appropriate, we remove this example and just go with the one he recommended.

Reply:
This looks a lot like my example below, so we should combine the two.

Note that the directories in this example all start with `/data/xrootdfs` because the paths are relative to `/`. There is no such prefix in my examples below -- the stashcache config I copied it from has `oss.localroot` defined so all paths are relative to that.

You must look at the xrootd config that gets installed to determine which examples are appropriate to keep. If the config has an `oss.localroot` line (or an `all.localroot` line perhaps?), you don't need a prefix. If it doesn't, then you do need a prefix.

Reply:
Perhaps we just keep your example and note that paths are relative to the XRootD config setting of `oss.localroot` or `/` if the config var is not set?

@matyasselmeci can you get back to us on if that includes `all.localroot` as well?

Reply:
They're pretty much the same thing. `oss.*` only affects the OSS subsystem and `all.*` affects all subsystems -- kinda like `SCHEDD.SCHEDD_NAME` versus `SCHEDD_NAME` in HTCondor config.

Reply:
Ok, @edquist your updated text should reference both `oss.localroot` and `all.localroot`

Reply:
Alright i think i've covered this section in my most recent updates.  (But let me know if it still needs work.)